content/how-to/hot-module-reload.md

```diff
@@ -0,0 +1,258 @@
+---
+title: How to Configure Hot Module Replacement?
```

Comment:
Cc @bebraw 

I think since there are many different HMR recipes, maybe we could structure this in like How-To -> HMR Recipies -> Babel/React/Redux

I think that way we encourage other recipes in here for many state libraries and frameworks. And it reinforces a much asked 'cookbook' for certain topics.